cmd/skaffold/app/cmd/flags.go

```diff
@@ -460,6 +460,22 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "Var",
 		DefinedOn:     []string{"test", "deploy"},
 	},
+	{
+		Name:          "auto-create-config",
```

Comment:
Could you set the `IsEnum` field to `true` for this flag and `assume-yes`?

Reply:
yep, done